dataprocspawner/spawner.py

```diff
@@ -1371,6 +1379,9 @@ def _build_cluster_config(self, cluster_data=None):
       cluster_data['config'].setdefault('software_config', {})
       cluster_data['config']['software_config'].setdefault('properties', {})
 
+      # If yaml exported from personal clusters, removes the personal auth info.
+      cluster_data['config']['software_config']['properties'].pop(prop_perso, None)
```

Comment:
Do we want to make it so that if the personal auth property is set in the config, we set it to the current user instead of dropping it entirely? I guess this comes to down to which behavior would be less surprising for admins.

Reply:
I was actually planning to do that with the original code, thinking that the `dataproc:dataproc.personal-auth.user` was `true` and not the user identity.

It's a good point. I will update.

Reply:
If the key is in the properties (whether added by user through allow_custom_clusters or because it was already in the YAML), the code now sets it to the user email.